### PR TITLE
Features missing from config should be disabled.

### PR DESCRIPTION
**Pull request by kdzwinel**
**Reviewer:** @jonathanKingston 

## Description:
When feature key is missing from the config we should disable the feature. This already got us in trouble once and we partially addressed it with https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/938 .

## Steps to test this PR:
1. Go to chrome-extension://hacmkdifffpphkffddioobolnmelljoa/html/list-editor.html in the extension
2. Change config to 

```json
{
  "readme": "https://github.com/duckduckgo/privacy-configuration",
  "version": 1636990486124,
  "features": {},
  "unprotectedTemporary": [
  ]
}
```

3. Make sure that all features, both content world (e.g. fingerprinting, 1p cookie expiry date thing) and background-page (e.g. GPC headers, content blocking) are disabled.

## Automated tests:
- [ ] Unit tests - unit test that checks this will be a follow up in https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/937
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/background/events.es6.js [hunk 1]
```diff
@@ -384,7 +384,8 @@ function getArgumentsObject (tabId, sender, documentUrl) {
     }
     // Special case for iframes that are blank we check if it's also enabled
     if (sender.url === 'about:blank') {
-        site.brokenFeatures = site.brokenFeatures.concat(utils.getBrokenFeaturesAboutBlank(tab.url))
+        const aboutBlankEnabled = utils.getEnabledFeaturesAboutBlank(tab.url)
+        site.enabledFeatures = site.enabledFeatures.filter(feature => aboutBlankEnabled.includes(feature))
```
Comment (jonathanKingston) [inline shared/js/background/events.es6.js:388]: Shouldn't this be:
```suggestion
        site.enabledFeatures = site.enabledFeatures.filter(feature => !aboutBlankEnabled.includes(feature))
```

Comment (kdzwinel) [reply shared/js/background/events.es6.js:388]: We want intersection of "features enabled in general" and "features enabled in about:blank iframes" right? If so this code looks good to me:

```js
const generalEnabled = ['a', 'b', 'c'];
const blankEnabled = ['b', 'c', 'd'];

generalEnabled.filter(feature => blankEnabled.includes(feature)) // results in ['b', 'c'] as expected
```

Comment (jonathanKingston) [reply shared/js/background/events.es6.js:388]: Agreed thanks :)

**Review by jonathanKingston — APPROVED** [review]
This is great :)
I always wanted the functions to be named in this manner too! (aka negative naming is a little harder to understand usually)

**Pull request merged**